experimental/add_to_app/SimpleAndroidApp/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="com.example.simpleandroidapp">
+
+    <application
+            android:name="io.flutter.app.FlutterApplication"
+            android:allowBackup="true"
+            android:icon="@mipmap/ic_launcher"
+            android:label="@string/app_name"
+            android:roundIcon="@mipmap/ic_launcher_round"
+            android:supportsRtl="true"
+            android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
+                <category android:name="android.intent.category.LAUNCHER"/>
+            </intent-filter>
+        </activity>
+        <activity
+            android:name="io.flutter.embedding.android.FlutterActivity"
+            android:launchMode="singleTop"
```

Comment:
It's probably safest to leave this as-is, but I'm not sure that `singleTop` is required any more. @xster can you provide a sanity check here? Given that `FlutterEngine` is now completely independent of `FlutterActivity`, I'm thinking you should be able to have as many `FlutterActivity`s as you'd like.....

Reply:
It should be physically impossible to launch two of these activities simultaneously in this app, so I can remove it for now. @xster, let me know what you'd prefer going forward.